modules/bootkube/resources/manifests/mao-config-aws.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  mao-config: |
+    apiVersion: v1
+    kind: machineAPIOperatorConfig
+    provider: aws
+    aws:
+      clusterName: ${cluster_name}
+      clusterID: ${cluster_id}
+      image: ${aws_ami}
```

Comment:
This field be `imageOverride`. 
Instead of installer telling MAO to use an AWS ami, installer should only be allowed to override internal logic.
MAO should be given enough information to make decision about the AWS ami internally so that we can depend on MAO to update the amis for new machines in a machineset.

Reply:
> This field should be `imageOverride`.

nit: Can't the docs for this property declare that it is optional and have it be called `image`.  All optional properties are overrides when the caller sets them; I don't think we have to include an `Override` suffix in all those names.